worldedit-core/src/main/java/com/sk89q/worldedit/command/ApplyBrushCommands.java

```diff
@@ -97,7 +98,7 @@ private void setApplyBrush(CommandParameters parameters, Player player, LocalSes
         double radius = requireNonNull(RADIUS.value(parameters).asSingle(double.class));
         RegionFactory regionFactory = REGION_FACTORY.value(parameters).asSingle(RegionFactory.class);
         BrushCommands.setOperationBasedBrush(player, localSession, Expression.compile(Double.toString(radius)),
-            new Apply(generatorFactory), regionFactory, "worldedit.brush.apply");
+            new ApplyRegion(generatorFactory), regionFactory, "worldedit.brush.apply");
```

Comment:
Why's this change made? .Apply should be removed from imports also

Reply:
Apply is deprecated in favor of ApplyRegion or ApplyLayer. ApplyRegion is more suitable in this scenario.